js-runtime/index.js

```diff
@@ -933,7 +933,7 @@ module.exports = {
         body: body.length > 0 ? body : undefined,
       });
       const rstatus = response.status
-      const rheaders = Object.entries(response.headers)
+      const rheaders = [...response.headers.entries()].map(kv => [kv[0] + '', kv[1] + ''])
```

Comment:
`response.headers` wasn't actually an object, it was a `Headers` type with some weirdness. Of course JS being JS it just succeeded at doing effectively nothing before.